recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
     - xorg-libxt
     - xz
     - zlib
+    - glib
```

Comment:
Do we need it on run too? Also do you need #161 merged first?
Last and not really important, just curious, what does `glib` brings to `imagemagick`?

Reply:
I think run exports deals with the run section. we can cleanup the run section later.

I would like #161 to be merged first since qt hasn't migrated yet. This is surely going to cause headaches soon, so I want to delay that pain as much as possible.

Not sure what glib brings other the ability to install imagemagick